### PR TITLE
Remove mentions of py3.5-3.6

### PR DESCRIPTION
**Pull request by kit1980**

### CODE REVIEW

**Review by malfet — APPROVED** [review]

File: .jenkins/pytorch/test.sh [hunk 1]
```diff
@@ -613,7 +613,4 @@ else
   test_custom_backend
   test_torch_function_benchmark
   test_benchmarks
-  if [[ "${BUILD_ENVIRONMENT}" == *linux-xenial-py3.6-gcc7-test* || "${BUILD_ENVIRONMENT}" == *linux-xenial-py3.6-gcc5.4-test* ]]; then
-    test_python_gloo_with_tls
```
Comment (malfet) [inline .jenkins/pytorch/test.sh:617]: Hmm, should we perhaps moove gloo test to some supported config?

Comment (kit1980) [reply .jenkins/pytorch/test.sh:617]: Probably. Right now it's not running anyway - no 3.6 workflows.

Comment (kit1980) [reply .jenkins/pytorch/test.sh:617]: I've added an issue https://github.com/pytorch/pytorch/issues/78398

**Pull request closed without merging**